tests/OSPSuite.Presentation.Tests/Importer/Presenters/ColumnMappingPresenterSpecs.cs

```diff
@@ -445,7 +446,10 @@ public void the_unit_is_set_properly(string oldUnitDescription, string newUnitDe
          sut.SetDescriptionForRow(_model);
 
          //Assert
-         (_model.Source as MappingDataFormatParameter).MappedColumn.Unit.SelectedUnit.ShouldBeEqualTo(shouldUpdate ? newUnitDescription : oldUnitDescription);
+         var mappingDataFormat = (_model.Source as MappingDataFormatParameter);
+         mappingDataFormat.MappedColumn.Unit.SelectedUnit.ShouldBeEqualTo(shouldUpdate ? newUnitDescription : oldUnitDescription);
+         if (shouldUpdate && newUnitDescription != "?")
+            mappingDataFormat.MappedColumn.Dimension.HasUnit(mappingDataFormat.MappedColumn.Unit.SelectedUnit).ShouldBeTrue();
```

Comment:
mappingDataFormat.MappedColumn : create a variable and use it. It's referenced 3 times. This is going to make the test easier to understand